config/samples/07-operator-context/pro.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: OperatorContext
+metadata:
+  name: local-pro-test
+spec:
+  secretRef:
+    namespace: tyk-operator-system
+    name: tyk-operator-conf
```

Comment:
Could provide an example secret resource to illustrate the config options.

Reply:
Or in comments:

```
kubectl create secret -n tyk-operator-system generic tyk-operator-conf \
  --from-literal "TYK_AUTH=${TYK_AUTH}" \
  --from-literal "TYK_ORG=${TYK_ORG}" \
  --from-literal "TYK_MODE=${TYK_MODE}" \
  --from-literal "TYK_URL=${TYK_URL}" \
  --from-literal "TYK_TLS_INSECURE_SKIP_VERIFY=true"
```

Something like this

Reply:
done 6d03096